src/app/onboarding-complete/page.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import Link from 'next/link';
+import { LogoImage, OuterDiv } from '@/app/onboarding/styles';
+import { BigLinkButton } from '@/components/Buttons';
+import CONFIG from '@/lib/configs';
+import COLORS from '@/styles/colors';
+import { CenteringDiv, Flex, SmallCard } from '@/styles/containers';
+import { H2Centered, P } from '@/styles/text';
+import IJPLogo from '~/public/images/ijp_logo_blue.webp';
+
+export default function Page() {
+  return (
+    <OuterDiv>
+      <Link href={CONFIG.homepage}>
+        <LogoImage $show width="205" alt="IJP Logo" src={IJPLogo} />
+      </Link>
+      <CenteringDiv>
+        <Flex $direction="column" $gap="40px" $align="center">
+          <SmallCard>
+            <Flex
+              $direction="column"
+              $justify="center"
+              $gap="40px"
+              $px="40px"
+              $minH="100%"
+            >
+              <H2Centered>You&apos;re all set!</H2Centered>
+              <P $align="center">
+                You can now submit interest applications for the available
+                listings.
```

Comment:
nit: from the screenshot it looks like the word listings is left hanging. do we want to add a line break, maybe before "the available listings"?
not sure if this will be resolved by a smaller browser size?

Reply:
yep! nice catch catherine.
i made it something like this
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/29067b7c-0d72-4075-a9b6-9765f1bf3399)